pepper-apis/src/main/java/org/broadinstitute/ddp/json/admin/participantslookup/ParticipantsLookupPayload.java

```diff
@@ -0,0 +1,19 @@
+package org.broadinstitute.ddp.json.admin.participantslookup;
+
+import javax.validation.constraints.NotEmpty;
+
+import com.google.gson.annotations.SerializedName;
+
+public class ParticipantsLookupPayload {
+
+    @SerializedName("query")
+    private String query;
+
+    public ParticipantsLookupPayload(@NotEmpty String query) {
```

Comment:
I don't think we need the annotation?
```suggestion
    public ParticipantsLookupPayload(String query) {
```

Reply:
Removed annotation